### PR TITLE
Fix a couple of resurrection bugs

### PR DESCRIPTION
**Pull request by titanous**
This fixes two bugs found while doing manual QA on restarts, see the commit messages for details.

Refs #235

### CODE REVIEW

File: script/install-flynn.tmpl [hunk 1]
```diff
@@ -98,6 +98,9 @@ main() {
     --bin-dir    "/usr/local/bin"
   ln -fs "/usr/local/bin/flynn-linux-amd64" "/usr/local/bin/flynn"
 
+  info "setting libvirt defaults"
+  echo "libvirt_uris='qemu:///system'" >> /etc/default/libvirt-bin
```
Comment (titanous) [inline script/install-flynn.tmpl]: This needs a grep guard, if run more than once it will add a line each time.

**Comment by lmars** [comment]
LGTM

**Comment by titanous** [comment]
@lmars added 47b3feae09ee370373052709db4b973acb6f28d3 to fix a discoverd race.

**Comment by lmars** [comment]
LGTM

**Pull request merged**